Index: b/rootfs/etc/dhcp/dhcpd.conf
===================================================================
--- a/rootfs/etc/dhcp/dhcpd.conf
+++ b/rootfs/etc/dhcp/dhcpd.conf
@@ -10,15 +10,15 @@
 ddns-update-style none;
 
 # option definitions common to all supported networks...
-option domain-name "example.org";
-option domain-name-servers ns1.example.org, ns2.example.org;
+#option domain-name "example.org";
+#option domain-name-servers ns1.example.org, ns2.example.org;
 
 default-lease-time 600;
 max-lease-time 7200;
 
 # If this DHCP server is the official DHCP server for the local
 # network, the authoritative directive should be uncommented.
-#authoritative;
+authoritative;
 
 # Use this to send dhcp log messages to a different log file (you also
 # have to hack syslog.conf to complete the redirection).
@@ -57,6 +57,18 @@
 #  max-lease-time 7200;
 #}
 
+# a subnet for the subnet we will offer.
+# dns servers are from ccc berlin and digitalcourage
+subnet 192.168.42.0 netmask 255.255.255.0 {
+  range 192.168.42.10 192.168.42.50;
+  option broadcast-address 192.168.42.255;
+  option routers 192.168.42.1;
+  default-lease-time 600;
+  max-lease-time 7200;
+  option domain-name "local";
+  option domain-name-servers 213.73.91.35, 85.214.20.141;
+}
+
 # Hosts which require special configuration options can be listed in
 # host statements.   If no address is specified, the address will be
 # allocated dynamically (if possible), but the host-specific information
