Index: stage3/rootfs/etc/dhcp/dhcpd.conf
===================================================================
--- stage3.orig/rootfs/etc/dhcp/dhcpd.conf
+++ stage3/rootfs/etc/dhcp/dhcpd.conf
@@ -10,15 +10,15 @@
 ddns-update-style none;
 
 # option definitions common to all supported networks...
-option domain-name "example.org";
-option domain-name-servers ns1.example.org, ns2.example.org;
+#option domain-name "example.org";
+#option domain-name-servers ns1.example.org, ns2.example.org;
 
 default-lease-time 600;
 max-lease-time 7200;
 
 # If this DHCP server is the official DHCP server for the local
 # network, the authoritative directive should be uncommented.
-#authoritative;
+authoritative;
 
 # Use this to send dhcp log messages to a different log file (you also
 # have to hack syslog.conf to complete the redirection).
