Index: b/rootfs/etc/default/isc-dhcp-server
===================================================================
--- a/rootfs/etc/default/isc-dhcp-server
+++ b/rootfs/etc/default/isc-dhcp-server
@@ -14,5 +14,5 @@
 
 # On what interfaces should the DHCP server (dhcpd) serve DHCP requests?
 #	Separate multiple interfaces with spaces, e.g. "eth0 eth1".
-INTERFACESv4=""
-INTERFACESv6=""
+INTERFACESv4="wlan1"
+INTERFACESv6="wlan1"
